### ISSUE TITLE
Make all theme pass WCAG AA

### ISSUE THREAD

**Issue by ghostbear** [tachiyomiorg/tachiyomi#5506]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/tag/v0.11.1)**.
- [X] I will fill out all of the requested information in this form.

### Describe your suggested feature

**Accent colors that don't pass WCAG AA**
Default (suggested color #146AFF) 
Blue (suggested color #5478A0)
Green Apple  (suggested color #188140)
Midnight Dusk (suggested color #E51065)
Strawberry Daiquiri (suggested color #E8173A)
Yotsuba (suggested color #D64000)

**Comment by arkon** [comment]
Are the suggestions taking into consideration both the light and dark backgrounds?

I'm sort of ignoring AMOLED since that's an accessibility nightmare already anyway.

**Comment by ghostbear** [comment]
The suggestions are for AMOLED and light background, I can look over them again and use Dark instead AMOLED

**Comment by Soitora** [comment]
Purely checking the contrast checker to adjust for both Light and Dark, this does not include in-app testing.

## **Default**

### Light
Default: #2979FF
Suggested: #146AFF
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

### Dark
Default: #2979FF
Suggested: #478BFF
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

## **Blue**

### Light
Default: #54759E
Suggested: #5478A0
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

### Dark
Default: #54759E
Suggested: #7391B5
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

## **Green Apple**

### Light
Default: #48E484
Suggested: #188140
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

### Dark
Default: #48E484
Suggested: #48E484
Passes all WCAG AAA requirements.

## **Midnight Dusk**

### Light
Default: #F02475
Suggested: #E51065
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

### Dark
Default: #F02475
Suggested: #F34F90
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

## **Strawberry Daiquiri**

### Light
Default: #ED4A65
Suggested: #E8173A
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

### Dark
Default: #ED4A65
Suggested: #EE5871
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

## **Yotsuba**

### Light
Default: #FC8C5C
Suggested: #D34203
Passes all WCAG AAA requirements, only passes WCAG AA for Normal Text.

### Dark
Default: #FC8C5C
Suggested: #FC9569
Passes all WCAG AAA requirements.

**Comment by Soitora** [comment]
> _Other comments:_ While it would be optimal to meet WCAG AAA for all cases where accent colors appear, that would be unreasonable since they also appear on manga covers and the covers are not a single universal background color. It would need some way to parse the color of the manga cover color and adjust the theme accent color accordingly

A way to have dynamic badge colors based on cover colors sounds like it *could* be a neat feature. Not sure how it would actually be
